src/VRPaths_openvr.cpp

```diff
@@ -0,0 +1,262 @@
+// NOTE: the following is an amalgamation of vrpathregistry_public.cpp, strtools_public.cpp, envvartools_public.cpp, and pathtools_public.cpp
+//       from openvr, to avoid the questions of "Can we already call this somehow?"
+//       or "How can we directly integrate this in our build system?"
+// TODO: Test on OSX/Linux, because our build system is probably not outputting the macros that this wants for them.
+//       I already had to change WIN32 to _WIN32 for the initial include of windows.h
+// TODO: This is licensed under BSD 3-Clause which is compatible with MIT,
+//       but we should probably do something to ensure we comply with clause 2.
```

Comment:
Clause 1 is already fulfilled because of the notice i left in source under this comment, but we may want to add a file to the driver containing license information if not already present to fulfill clause 2, or add something to the server UI mentioning libraries used by various components.

I think something similar also applies to the simdjson library I'm using here. It's Apache 2, which should also be compatible, but similarly requires a license notice.